src/Service/ValueService.php

```diff
@@ -176,7 +176,7 @@ public function findSubobject(string $identifier, Value $valueObject): ?ObjectEn
     {
         if (Uuid::isValid($identifier) === true) {
             return $this->getSubObjectById($identifier, $valueObject);
-        } else if (filter_var($identifier, FILTER_VALIDATE_URL) === true) {
+        } else if (filter_var($identifier, FILTER_VALIDATE_URL) !== false) {
```

Comment:
Why change this?

Reply:
because the return value of filter_var is either false if invalid, or the string itself.